content/en/docs/getting-started/setup/_index.md

```diff
@@ -1,8 +1,12 @@
 ---
-title: "Setup Jenkins X"
-linkTitle: "Setup Jenkins X"
+title: Setup
+linkTitle: Setup
 weight: 1
 description: >
   How to setup Jenkins X?
 ---
 
+Setting up Jenkins X consist of three steps.
+First you need to create and prepare a Kubernetes cluster for the installation of Jenkins X.
+Next you need to install the `jx` command line interface.
+Last but not least, you use `jx` to run `jx boot`.
```

Comment:
```suggestion
Finally, use `jx` command to run `jx boot`.
```

Reply:
What's about "Finally, use the `jx` CLI to run `jx boot`"?